### ISSUE TITLE
vol-tests can't be built without szip when HDF5_VOL_TEST_ENABLE_PARALLEL is on.

### ISSUE THREAD

**Issue by hyoklee** [HDFGroup/vol-tests#8]
@jhendersonHDF , Is SZIP required for parallel test?
If so, please update README.md.

**Comment by jhendersonHDF** [comment]
No, SZIP is not required for the parallel tests to be built. All code that uses szip-related calls is surrounded by

    H5_HAVE_FILTER_SZIP

and will not be built unless the HDF5 installation reports that SZIP is available. This suggests that there is a problem with the HDF5 installation being used.
